packages/sdk/src/modules/liquidation/redemptionStrategy.ts

```diff
@@ -208,24 +231,26 @@ const getStrategyAndData = async (fuse: MidasBase, inputToken: string): Promise<
 
       return { strategyAddress: redemptionStrategyContract.address, strategyData, outputToken };
     }
-    default: {
-      return { strategyAddress: redemptionStrategyContract.address, strategyData: [], outputToken };
-    }
-  }
-};
+    case RedemptionStrategyContract.BalancerLpTokenLiquidator: {
+      // TODO: do we need this at all? Or can we just use the output token directly?
```

Comment:
@vminkov double check if we need this logic at all, or if we rely on the `outputToken` that is set in the `redemptionStrategies.ts`